tests/integration/test_pip.py

```diff
@@ -53,15 +53,10 @@
         ),
         pytest.param(
             utils.TestParameters(
-                branch="pip/multiple-packages",
+                branch="pip-multiple-devel",
```

Comment:
Agree in principle, I guess this is just to showcase, right? We need to simplify the upstream testing branch accordingly.